Documentation/Metrics/rocksdb_cache_edge_inserts_effective_entries_size_total.yaml

```diff
@@ -1,20 +1,22 @@
-name: rocksdb_cache_edge_effective_entries_size
+name: rocksdb_cache_edge_inserts_effective_entries_size_total
+renamedFrom: rocksdb_cache_edge_effective_entries_size
 introducedIn: "3.11.2"
 help: |
   Total effective memory size of all edge cache entries ever stored in the edge cache.
 unit: bytes
-type: gauge
+type: counter
 category: Statistics
 complexity: advanced
 exposedBy:
   - dbserver
   - agent
   - single
 description: |
-  Total effective memory size of all edge cache data that were ever stored in
-  the edge cache of any collection/shard. The size is calculated after any
-  potential compression, so the compression efficiency can be estimated by 
-  this metric by the size of the uncompressed edge cache entries.
+  Total effective memory size of all edge cache data that were stored in the
+  the edge cache of any collection/shard since the server start. 
```

Comment:
NIT counters are not reset (with the help of prometheus) so this is actually _since the server initially started_.